packages/hoc-withtooltip/src/withTooltip.js

```diff
@@ -0,0 +1,101 @@
+import * as React from "react";
+import Tooltip, { alignments } from "@crave/farmblocks-tooltip";
+import wrapDisplayName from "recompose/wrapDisplayName";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import values from "object.values";
+
+const Container = styled.div`
+  position: relative;
+  display: ${props => (props.displayBlock ? "block" : "inline-block")};
+
+  .hitArea {
+    position: absolute;
+    top: 0;
+    width: 100%;
+    height: 100%;
```

Comment:
This element is covering the wrapped component. Click events and hover styles are not been triggered.

Check the `HOC disabledTooltip - disabled false and button events (click, mouseover)` story.